config/main.py

```diff
@@ -1999,6 +2002,20 @@ def override_config_db(config_db, config_input):
     click.echo("Overriding completed. No service is restarted.")
 
 
+def yang_hard_dependency_check(config_json):
+    aaa_yang_hard_dependency_check(config_json)
+
+
+def aaa_yang_hard_dependency_check(config_json):
+    AAA_TABLE = config_json.get("AAA", {})
```

Comment:
>AAA [](http://example.com/codeflow?start=33&length=3)

Not a hard blocker. Is it possible that yang-model could describe the dependency? Is there or will there be a PR to fix the yang-model?

Reply:
I don't find a YANG model way to describe the dependency.
There will be a fix for yang model.